docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,486 @@
+                <span class="toggle-button__image-container">
+                    <span class="toggle-button__image">
+                        <img src="{{ page.static_dir }}/img/tb-profile-pic.jpg" alt="">
+                    </span>
+                </span>
+                <span class="toggle-button__content">
+                    <span class="toggle-button__title">260px Sized Button</span>
+                    <span class="toggle-button__subtitle">Subtitle</span>
+                </span>
+            </button>
+
+            <button type="button" class="toggle-button toggle-button--gallery-layout" aria-pressed="false" style="width: 342px;">
+                <span class="toggle-button__image-container">
+                    <span class="toggle-button__image">
+                        <img src="{{ page.static_dir }}/img/tb-profile-pic.jpg" alt="">
+                    </span>
+                </span>
+                <span class="toggle-button__content">
+                    <span class="toggle-button__title">342px Sized Button</span>
+                    <span class="toggle-button__subtitle">Subtitle</span>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button type="button" class="toggle-button toggle-button--gallery-layout" aria-pressed="false" style="width: 168px;">
+    <span class="toggle-button__image-container">
+        <span class="toggle-button__image">
+            <img src="{{ page.static_dir }}/img/tb-profile-pic.jpg" alt="">
+        </span>
+    </span>
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">Minimum Sized Button</span>
+        <span class="toggle-button__subtitle">Subtitle</span>
+    </span>
+</button>
+<button type="button" class="toggle-button toggle-button--gallery-layout" aria-pressed="false" style="width: 260px;">
+    <span class="toggle-button__image-container">
+        <span class="toggle-button__image">
+            <img src="{{ page.static_dir }}/img/tb-profile-pic.jpg" alt="">
+        </span>
+    </span>
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">260px Sized Button</span>
+        <span class="toggle-button__subtitle">Subtitle</span>
+    </span>
+</button>
+<button type="button" class="toggle-button toggle-button--gallery-layout" aria-pressed="false" style="width: 342px;">
+    <span class="toggle-button__image-container">
+        <span class="toggle-button__image">
+            <img src="{{ page.static_dir }}/img/tb-profile-pic.jpg" alt="">
+        </span>
+    </span>
+    <span class="toggle-button__content">
+        <span class="toggle-button__title">342px Sized Button</span>
+        <span class="toggle-button__subtitle">Subtitle</span>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <h4>Responsive Images Using CSS Background-Image</h4>
```

Comment:
Why don't these demo blocks have `{% highlight html %}` sections? I don't think they are _absolutely_ necessary, but it does seem a little bit inconsistent.

Reply:
I can quickly add those. good catch.